### DISCUSSION TITLE
Idea on how to fix chrome support

### DISCUSSION THREAD

**Discussion by Spade-775** [lightspark/lightspark#673] [Ideas]
So I have heard that lighspark don't work yet with chrome and I may have found why. when I went to use chrome://extensions/ in developer mode to install it using "load unpacked" on the PepperFlash, I found what wasn't working, it was missing **"manifest_version": 2**, which is a key command that is required for it to work with chrome (recommended to place below **"version": "50.0.8.0",** ), add this and everything should work without a problem!

**Comment by dbluelle** [comment]
Have you tested that, does it work?
AFAIK the flash plugin is not handled as a "normal" extension, so it would not be installed via chrome:extensions

**Reply by Spade-775** [reply]
I am unsure as I am not the best at navigating source codes, And I've been unable to edit the file in the official download So I cannot test it.

**Reply by dbluelle** [reply]
Hmm, the manifest.json of the official download should be editable, I don't know much about windows, but maybe you have to edit it with administrator rights.

**Reply by Spade-775** [reply]
Sorry my reply is late but no; "only the original developers can edit this file" probably some windows anti hacker thing

**Reply by dbluelle** [reply]
Hmm, thats bad, I'd rather have someone test if this really works before making the change in the repository.

**Reply by Spade-775** [reply]
I can understand that, hope you find some one that can.

**Reply by veso266** [reply]
It should be theoretically possible (at least until June 2022 when they kill PPAPI according to this: https://techcommunity.microsoft.com/t5/discussions/good-news-google-is-finally-killing-off-chrome-apps-end-support/m-p/1112057

BTW: Can someone send me some Chrome App (wasn't able to find any, and can't diferenciate between extensions)

**Reply by Spade-775** [reply]
It's in the Lightpark files under the folder PepperFlash, both extention files for google should be there. ( Programfiles/lightspark/PepperFlash/ )

**Reply by veso266** [reply]
Got it installed but got some errors after: Unrecognized manifest key 'x-ppapi-required-interfaces'.

@phantomDragon533  you probably edited the file without admin rights as on windows you need them if you want to edit files in program files

I copied the dir in different location and got it installed
![slika](https://user-images.githubusercontent.com/6347325/114775619-c7918480-9d71-11eb-87e6-2ea33b309f1b.png)
![slika](https://user-images.githubusercontent.com/6347325/114775587-c06a7680-9d71-11eb-9ce4-e8af4784cdf7.png)

but now I need to figure out what x-ppapi-required-interfaces is 

even tried original flash player (the last one 32.0.0.465, patched to remove the timebomb)
and it came up with even more errors

keys in manifest giving problems are
```
"x-flapper-revision": "104123",
"x-ppapi-arch": "x64", 
"x-ppapi-os": "win", 
```


![slika](https://user-images.githubusercontent.com/6347325/114776606-f4926700-9d72-11eb-94dc-65aafbd70789.png)

**Reply by Spade-775** [reply]
I am no expert in programming and I myself have been unable to find out what it is, It probably doesn't recognize the input for X-ppapi and requires different keys, but I did find some things about fake errors with these plugs so it mite be that as well.

**Reply by ritsika** [reply]
heyy, were you able to fix that error?

**Reply by huwdp** [reply]
There's a page here (https://github.com/lightspark/lightspark/wiki/Getting-Lightspark-up-and-running-in-Web-Browser).  on how to get Lightspark working under various web browsers
